readme.md

```diff
@@ -96,6 +96,7 @@ All the below methods accept a value and returns a boolean for whether the value
 
 Keep in mind that [functions are objects too](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions).
 
+##### .numericString(value)
```

Comment:
There should be a description here, as it's not immediately clear what a `numericString` entails.